build/tools/amebasmart/gnu_utility/loadable_xip_elf.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+###########################################################################
+#
+# Copyright 2024 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+import os
+
+filepath = os.path.dirname(__file__) + "/km0_km4_app.bin"
+alignment = 4096
+
+def get_offset():
+    bin_size = os.path.getsize(filepath)
+    # Binary need to be aligned to 4K
+    bin_size = (bin_size + alignment - 1) // alignment * alignment
+    # 4K padding before AP binary (ie. running on Tizen Lite) starts
+    bin_size = bin_size + alignment
+    return hex(bin_size)
```

Comment:
I have a minor question. Does this also handle the size of Samsung header that we add at the start of the trpk file? If it doesn't then we will have issues at some cases where the alignment goes wrong.

Reply:
Yes, the km0_km4 binary has a 4kb of header which is same as trpk file, despite of the values. Which means calculating from km0_km4 will yield the correct result.
In detail: 4KB header -> 4KB km4_km0 manifest -> km4_km0 image size with 4KB alignment -> 4KB ca32 manifset
Thus, your concern should be handled in the above logic.

Reply:
Okay, thanks for explaining

Reply:
If there is no further question, could you leave an approval? Thanks~